src/Microsoft.ML.KMeansClustering/KMeansPlusPlusTrainer.cs

```diff
@@ -91,7 +91,7 @@ internal static class Defaults
 
         /// <summary>
         /// Options for the <see cref="KMeansTrainer"/> as used in
-        /// [KMeansTrainer(Options)](xref:"Microsoft.ML.KMeansClusteringExtensions.KMeans(Microsoft.ML.ClusteringCatalog.ClusteringTrainers,Microsoft.ML.Trainers.KMeansTrainer.Options)".
```

Comment:
>xref [](start = 37, length = 4)

xref does work (see for example in `SgdCalibratedTrainer.Options`). I think the quotes are messing it up, and also there is a close parentheses that is missing.

Reply:
I am not sure. I have seen xref are not rendered correctly in other places. After discussing Arti and Ivan, we feel ok to use cref because user will still be lined to the right page.

---
In reply to: [277886641](https://github.com/dotnet/machinelearning/pull/3550#discussion_r277886641) [](ancestors = 277886641)

Reply:
please don't change xrefs where they are used to shorten the anchor text.

Yael pointed out the issues: delete quotes, add missing ) at the end.

if you paste UID from xref service directly into [anchor](xref:UID) you shouldn't have rendering issues.



---
In reply to: [277888214](https://github.com/dotnet/machinelearning/pull/3550#discussion_r277888214) [](ancestors = 277888214,277886641)

Reply:
Ok. Fixed.

---
In reply to: [277889265](https://github.com/dotnet/machinelearning/pull/3550#discussion_r277889265) [](ancestors = 277889265,277888214,277886641)